lib/mobility/arel.rb

```diff
@@ -4,7 +4,25 @@
 
 module Mobility
   module Arel
+    module MobilityExpressions
+      include ::Arel::Expressions
+
+      def as(*)
+        super
+          .extend(::Arel::Expressions)
```

Comment:
This is necessary due to [this line](https://github.com/shioyama/rails/blob/847342c25c61acaea988430dc3ab66a82e3ed486/activerecord/lib/active_record/relation/calculations.rb#L261). We might not need it though.

Reply:
Seems we do need it.